### PR TITLE
Drop the custom "remember me" implementation

### PR DESCRIPTION
**Pull request by aschempp**
Manually tested this locally:
 - `contao:migrate` correctly generates the schema
 - the cookie name is still the same
 - deleting the session cookie logs me back in through rememberme
 - if I manually log out, the remember me cookie and database get cleared

### CODE REVIEW

**Review by Toflar — DISMISSED** [review]

**Comment by ausi** [comment]
Do we need a migration that copies the data from the old into the new format?

**Comment by Toflar** [comment]
No, we agreed that data can be dropped.

**Review by leofeyer — DISMISSED** [review]

**Comment by aschempp** [comment]
@leofeyer I intentionally did not change the test config (https://github.com/contao/contao/pull/6815/commits/e152b718e59b9bd6a917c28492afc869f8661b06) because our token authenticator was not configured! Now tests would use a doctrine DB instead of stateless rememberme.

**Comment by leofeyer** [comment]
A comment would be good so we remember this next time. 😉

**Comment by aschempp** [comment]
Added migration and manually tested it:
 - checkout 5.3.x-dev
 - login as d.evans in the demo, including autologin checkbox. A session and a rememberme cookie is created
 - checkout this branch and run the migration
 - drop the session cookie
 - d.evans is still logged in (because the existing cookie is now looked up in the new table)

File: core-bundle/config/migrations.yaml [hunk 1]
```diff
@@ -70,3 +70,9 @@ services:
         arguments:
             - '@database_connection'
             - '@contao.framework'
+            -
```
Comment (ausi) [inline core-bundle/config/migrations.yaml]:
```suggestion

```
this empty parameter looks like a mistake?

**Review by ausi — DISMISSED** [review]
Approved apart from https://github.com/contao/contao/pull/6815#discussion_r1471173169

**Review by ausi — DISMISSED** [review]

**Review by leofeyer — DISMISSED** [review]

**Comment by leofeyer** [comment]
Thank you @aschempp.

**Review by leofeyer — APPROVED** [review]

**Pull request merged**